### ISSUE TITLE
ChecklistAugmenter AssertionError

### ISSUE THREAD

**Issue by Cerebrock** [QData/TextAttack#638]
**Describe the bug**
ChecklistAugmenter is bugged, error is
``AssertionError: Unexpected type of parameter 'use_tokenizer'. Parameter should be bool or Tokenizer```

**To Reproduce**
Run Augmentation example notebook: 
https://colab.research.google.com/github/QData/TextAttack/blob/master/docs/2notebook/3_Augmentations.ipynb

**Comment by jpilaul** [comment]
I've changed the init params of `Sentence` in `attacked_text.py` 
From:
```python
sentence = Sentence(
    self.text, use_tokenizer=textattack.shared.utils.words_from_text
)
```
To:
```python
sentence = Sentence(
    textattack.shared.utils.words_from_text(self.text), use_tokenizer=False
)
```
And it works like a charm

**Comment by VijayKalmath** [comment]
From PR #293-(Oct 6, 2020) , we send the function textattack.shared.utils.words_from_text for use_tokenizer in the pos_of_word_index function. 

    def pos_of_word_index(self, desired_word_idx):
        """Returns the part-of-speech of the word at index `word_idx`.
        Uses FLAIR part-of-speech tagger.
        """
        if not self._pos_tags:
            sentence = Sentence(
                self.text, use_tokenizer=textattack.shared.utils.words_from_text   <<< Function sent as Callable function.
            )
            textattack.shared.utils.flair_tag(sentence)
            self._pos_tags = sentence
        flair_word_list, flair_pos_list = textattack.shared.utils.zip_flair_result(
            self._pos_tags
        )


And as of PR "Major refactoring of internal label logic #2645" in the flairNLP Repo (Feb 11, 2022) in  "https://github.com/flairNLP/flair/blob/master/flair/data.py" - Flair does not allow callable functions as input for the Sentence Class.

To follow Flair's implementation and fix the errors seen in Issues #642 and #638, we should revert PR "have flair use textattack tokenization method #293". 

I can go ahead and submit a PR to revert PR #293 if it makes overall sense.

**Comment by Jebrankhan** [comment]
Can you please revert #293 if there is no other way to resolve #642 and #638?
